crates/protocol/derive/src/metrics/mod.rs

```diff
@@ -25,6 +25,12 @@ impl Metrics {
 
     /// Identifier for the channel buffer memory overhead gauge.
     pub const PIPELINE_CHANNEL_MEM: &str = "kona_derive_channel_mem";
+
+    /// Identifier for a gauge that tracks the number of blocks until the next channel times out.
+    pub const PIPELINE_CHANNEL_TIMEOUT: &str = "kona_derive_channel_timeout";
```

Comment:
Perhaps we can use a more descriptive name? `kona_derive_blocks_until_channel_timeout` or something (verbose, but describes what's being tracked a bit better - I read this as a static value.)

Reply:
Yea sure a bit picky but happy to increase verbosity